src/flowchem/devices/runze/runze_valve_component.py

```diff
@@ -24,12 +24,13 @@ def __init__(self, name: str, hw_device: RunzeValve) -> None:
         self.add_api_route("/monitor_position", self.get_monitor_position, methods=["GET"])
         self.add_api_route("/monitor_position", self.set_monitor_position, methods=["PUT"])
 
-    def _change_connections(self, raw_position: str, reverse: bool = False):
-        raw_position = int(raw_position)
+    def _change_connections(self, raw_position: str | int, reverse: bool = False):
+        position_mapping = {0: 4, 1: 3, 2: 2, 3: 1, 4: 6, 5: 5}
```

Comment:
would it become slightly easier with eg 
anti_cc_stator = [4, 3, 2, 1, 6, 5]
if reverse:
    return anti_cc_stator[raw_position]
else:
    return anti_cc_stator.index(raw_position)

Main benefit: instead of supplying the mapping, you just write down the valves internal port numbers by going, following from top clockwise around the valve